src/js/background.js

```diff
@@ -217,7 +217,6 @@ browser.storage.onChanged.addListener(updatePasswordMenu);
 let contextMenuAccounts = [];
 
 function updatePasswordMenu() {
-    if(isChrome && !browser.hasOwnProperty('contextMenus') || !isChrome && !browser.hasOwnProperty('menus')) return;
```

Comment:
You can't just delete things :smile: That is here to prevent the context menu if the api is missing (Firefox Android). I fixed the bug with chrome in the master branch.

Reply:
You added that line of code recently. I though deleting it will be ok. 😄
Sorry for that. 

If you can reject that file and accept translation. Do it that way. I will work contextmenu on it in my free time.


-----------
And can you update building scripts? Or am i missing something idk. I cant build and test.

```
npm install
npm run build:chrome
```
not worked for me

Thanks.

Reply:
I'm not aware of any issues with the build script. Can you provide details?

Reply:
build is supposed to be an environment variable.

I have changed it so that the platform is now defined as a parameter.

Reply:
Thanks mate. 👍